ssl_cert_tracker/tests/test_db_helper.py

```diff
@@ -0,0 +1,41 @@
+"""
+.. _models:
+
+django models for the ssl_certificates app
+
+:module:    test_db_helper.py
+
+:copyright:
+
+    Copyright 2018 Provincial Health Service Authority
+    of British Columbia
+
+:contact:    ali.rahmat@phsa.ca
+
+"""
+import pytest
+from django.utils import timezone
+from ssl_cert_tracker.models import NmapCertsData, NmapHistory
+
+@pytest.mark.django_db
+def test_insert_into_certs_data():
+    """test_check_tag_valid reads a valid xml file

```

Comment:
what are you testing here? based on the assert statements, you are testing that the 2 different attributes originating from a known data source are different which is rather obvious. you are also directly testing that django queryset filter() and django queryset count() methods are correct which the django people have already tested many, many times. indirectly, you are testing that the django.db.models.Model.save() method works which the django people have also tested many, many time.

Reply:
I am pushing a record into model
then I check the count making sure the record is inserted....
The test may look obvious but it is sanity check